main.py

```diff
@@ -1 +1,74 @@
-pass
+import re
+import datetime
+from collections import OrderedDict
+
+def create_racer_abbreviations_dict(file_name):
+    """Retrieves {'abbreviation': (name, team)}" format dict from abbreviations.txt"""
+    abbreviations_dict = {}
+    with open(file_name, 'r') as fn:
+        for line in fn:
+            matchObj = re.match(r'^(\w+)_([a-zA-Z\s]+)_([a-zA-Z\s]+)$', line)
+            # group(1) is abbreviation, i.e 'SVM'
+            abbreviations_dict[matchObj.group(1)] = (
+                matchObj.group(2),  # name of a pilot
+                matchObj.group(3).rstrip(),  # team
+            )
+    return abbreviations_dict
+
+
+# {'abbreviation of pilot': ('name of pilot, 'team')}
+abbr_dict = create_racer_abbreviations_dict(
```

Comment:
lines more then 110 (can wary belong to project from 100 to 120) symbols must be separated on smaller parts according to PEP8; All code beside the function and classes recommended to have inside of 
`if __name__ == "__main__":` block at the end of the file;